assets/downloads.css

```diff
@@ -32,15 +32,18 @@ img {
   margin-bottom: 10px;
 }
 
-.card__right {
+#lawnchair_title {
+  padding-top: 10px;
+}
+
+#card_main_cta {
```

Comment:
Can't you simply use `.card__main h2` and `.card__main .card__right`?

Reply:
Correct. Will replace in next commit.